assembl/static2/js/app/components/administration/voteSession/voteProposalForm.jsx

```diff
@@ -116,6 +116,10 @@ const DumbVoteProposalForm = ({
     displayCustomModal(content, true, 'gauge-settings-modal');
   };
 
+  const isTitleEmpty = title === '' || title === null;
+  const isDescriptionEmpty = description === null || description.blocks.every(block => block.text === '');
```

Comment:
title & description may not be undefined ?

Reply:
you should probably use `rawContentStateIsEmpty` (in `utils/draftjs.js`) that will work if there is an atomic block in your raw content state

Reply:
for title & description they are set to null before anything is typed in, do you see a case where they would turn to undefined ? 
I'll check rawContentStateIsEmpty ;)

Reply:
no, it should be ok